blog/2021-03-16-BatchSolvingWithAmq.adoc

```diff
@@ -0,0 +1,182 @@
+= Batch solving in a stateless manner with OptaPlanner and AMQ
+:page-interpolate: true
+:imagesdir: 2021/03/16
+:awestruct-author: rsynek
+:awestruct-layout: blogPostBase
+:awestruct-tags: [amq, quarkus, messaging, integration]
```

Comment:
This creates two new tags: amq and messaging.
Do we expect to write 2 more blog posts for these tags in this year? Tags with one article aren't that useful I'd argue.

I'd remove messaging. I do expect "amq" (like kafka) to come back often:
```suggestion
:awestruct-tags: [amq, quarkus, integration]
```